Packs/SilentPush/Integrations/SilentPush/SilentPush.py

```diff
@@ -148,36 +417,319 @@ def test_module(client: Client) -> str:
 ''' COMMAND FUNCTIONS '''
 
 
-def list_domain_information_command(client: Client, args: dict) -> CommandResults:
+def list_domain_information_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    domains_arg = args.get('domains', args.get('domain'))
+    if not domains_arg:
+        raise DemistoException('No domains provided. Please provide domains using either the "domain" or "domains" argument.')
+
+    if isinstance(domains_arg, str):
+        domains = [d.strip() for d in domains_arg.split(',')]
+    else:
+        domains = domains_arg
+
+    if len(domains) > 100:
+        raise DemistoException("Maximum of 100 domains can be submitted in a single request")
+
+    demisto.debug(f'Processing domains in bulk: {domains}')
+    raw_response = client.list_domain_information(domains)
+    demisto.debug(f'Response from API: {raw_response}')
+
+    markdown = ['# Domain Information Results\n']
+
+    for domain_info in raw_response.get('domains', []):
+        markdown.append(f"## Domain: {domain_info.get('domain')}")
+
+      
+        basic_info = {
+            'Created Date': str(domain_info.get('whois_created_date', 'N/A')),
+            'Registrar': str(domain_info.get('registrar', 'N/A')),
+            'Age (days)': str(domain_info.get('age', 'N/A')),
+            'Risk Score': str(domain_info.get('sp_risk_score', 'N/A'))
+        }
+        markdown.append(tableToMarkdown('Domain Information', [basic_info]))
+
+        
+        risk_explain = str(domain_info.get('sp_risk_score_explain', 'N/A'))
+        if risk_explain != 'N/A':
+            markdown.append(f'### Risk Score Explanation\n{risk_explain}')
+
+     
+        whois_info = domain_info.get('whois_info', {})
+        if whois_info:
+            whois_info_list = [{'Key': k, 'Value': v} for k, v in whois_info.items()]
+            markdown.append(tableToMarkdown('WHOIS Information', whois_info_list))
+
+        markdown.append('\n---\n')  
+
+    return CommandResults(
+        outputs_prefix='SilentPush.Domain',
+        outputs_key_field='domain',
+        outputs=raw_response.get('domains', []),
+        readable_output='\n'.join(markdown),
+        raw_response=raw_response
+    )
+
+
+
+def get_domain_certificates_command(client: Client, args: dict) -> CommandResults:
+    """
+    Command handler for fetching domain certificate information.
     """
-    Command handler for fetching domain information.
+    domain = args.get('domain', 'silentpush.com')
```

Comment:
```suggestion
    domain = args.get('domain', 'silentpush.com')
```
@karand-metron Please remove hard coded values